templates/studentAffectedByModule.html

```diff
@@ -0,0 +1,16 @@
+$def with (moduleCode, aysem, student_lst)
```

Comment:
![image](https://cloud.githubusercontent.com/assets/12403982/24074518/4ffda85c-0c45-11e7-8ed2-39198bc60ca1.png)
look for cs1010